app/controllers/works_controller.rb

```diff
@@ -0,0 +1,89 @@
+class WorksController < ApplicationController
+  before_action :set_work, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @books = Work.get_in_category_by_votes("book")
+    @movies = Work.get_in_category_by_votes("movie")
+    @albums = Work.get_in_category_by_votes("album")
+  end
+
+  def show
+    @users = []
+    @work.votes.each do |vote|
+      user = vote.user
+      @users << user
+    end
+  end
+
+  def new
+    @work = Work.new
+  end
+
+  def edit; end
+
+  def create
+    @work = Work.new(work_params)
+
+    if @work.save
+      flash[:success] = "Successfully created #{@work.category} #{@work.id}"
+      redirect_to @work
+    else
+      flash[:error] = "A problem occurred: Could not create #{@work.category}"
+      flash[:error_messages] = @work.errors.messages
+      render :new
```

Comment:
You should also include a status code

```suggestion
      render :new, status: :bad_request
```